packages/mmdb/mmdb.0.2.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+homepage: "https://issuu.github.io/ocaml-mmdb"
+dev-repo: "git+https://github.com/issuu/ocaml-mmdb.git"
+bug-reports: "https://github.com/issuu/ocaml-mmdb/issues"
+doc: "https://issuu.github.io/ocaml-mmdb/"
+license: "Apache-2.0"
+maintainer: "Team Platypus <platypus@issuu.com>"
+build: [["dune" "build" "-p" name "-j" jobs]]
+run-test: [["dune" "runtest" "-p" name "-j" jobs]]
+depends: [
+  "alcotest" {with-test & >= "0.8.5"}
+  "base" {>= "v0.10"}
+  "conf-libmaxminddb"
+  "ctypes" {>= "0.14"}
+  "ctypes-foreign" {>= "0.4"}
+  "dune" {build & >= "1.6"}
+  "ocaml" {>= "4.04.1"}
+  "ppx_deriving" {build & >= "4.2"}
```

Comment:
The ppxs may have a runtime (now or in the future), they should not be marked as build in general

Reply:
I'm gonna close this PR and open another one with version 0.2.1 where this is fixed.